src/herder/TransactionQueue.cpp

```diff
@@ -117,6 +117,11 @@ canReplaceByFee(TransactionFrameBasePtr tx, TransactionFrameBasePtr oldTx,
         {
             minFee = INT64_MAX;
         }
+        else
+        {
+            // Add the potential flat component to the resulting min fee.
+            minFee += tx->getFullFee() - tx->getFeeBid();
```

Comment:
Wouldn't it make more sense to use the full fee instead of the fee bid for `newFee` and `oldFee`? The math here is a little weird since it doesn't take the `FEE_MULTIPLIER` into account. I'm not familiar with how minFee is used, but I'm assuming that would lead to the most accurate value?

Reply:
`minFee` is just a rough estimate of how much you need to bid to get included - given that there is a constant stream of transactions it can't be too accurate. Using full fee is not correct for the sake of comparison though - only bids are used to make the inclusion decision (here we would return 10x the bid), but the flat part will remain flat.